lib/minitest/reporters/mean_time_reporter.rb

```diff
@@ -153,17 +153,21 @@ def order_sorted_body
       # @return [Array<Hash<Symbol => String>>] All of the results sorted by
       #   the :sort_column option. (Defaults to :avg).
       def column_sorted_body
-        previous_run.each_with_object([]) do |(description, timings), obj|
+        current_suite_names = self.tests.each_with_object(Set.new) { |suite, set| set << suite.class.name }
+        previous_run.each_with_object([]) do |(suite_name, timings), obj|
           size = Array(timings).size
           sum  = Array(timings).inject { |total, x| total + x }
           obj << {
             avg:  (sum / size).round(9).to_s.ljust(12),
             min:  Array(timings).min.round(9).to_s.ljust(12),
             max:  Array(timings).max.round(9).to_s.ljust(12),
             last: Array(timings).last.round(9).to_s.ljust(12),
-            desc: description,
+            desc: suite_name,
+            suite_name: suite_name
```

Comment:
I was thinking description can be changed in the future. It can have shorter test names or can have decorated test names.
Name `suit_name` is for clarity, when we see variable suite_name we exactly know what it is.

Anyway, I can use `description` to filter out suits, and will if you want.